config/vufind/RateLimiter.yaml

```diff
@@ -130,3 +143,17 @@ Policies:
       policy: token_bucket
       limit: 2000
       rate: { interval: '10 minutes', amount: 400 }
+
+# Cloudflare Turnstile is a cloud-based "CAPTCHA alternative" to detect bots.
+# https://developers.cloudflare.com/turnstile/
+# These are the global settings for Turnstile. See also the turnstile config above:
+# - within each policy, to enable turnstile for that policy
+# - storage settings for the result cache
+# Turnstile:
+  # These two keys are required. See also values they can be set to for testing purposes:
+  # https://developers.cloudflare.com/turnstile/troubleshooting/testing/
+  # siteKey: 0x1234567890
+  # secretKey: 0x1234567890
+
+  # Verify API URL.  Default is https://challenges.cloudflare.com/turnstile/v0/siteverify
+  # verifyUrl: https://challenges.cloudflare.com/turnstile/v0/siteverify
```

Comment:
It feels silly to write the URL twice here and again as the default in the code, when honestly I'm not sure it should be configurable in the first place (if the URL changes, there's likely some logic within that will also change).  Remove?

Reply:
I think it's worth making this configurable. I could imagine region-specific URLs or even something like an open source "work-alike" service that implements the same API. Probably not _likely_, but it doesn't hurt to have an easy configuration point just in case, so if the situation arises, we don't have to do any work to react to it. That being said, if you don't want to include the URL twice in the configuration, you could say "The example below shows the default value" (or something like that).

Reply:
That make sense...but if I'm planning for a work-alike service, would it also share the JS https://challenges.cloudflare.com/turnstile/v0/api.js loaded in the template?  I'm thinking I should make that configurable as well.

Reply:
Done.